<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IEEE80211g</class>
 <widget class="QMainWindow" name="IEEE80211g">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1281</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IEEE80211g</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>1011</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBoxPosX">
           <property name="minimum">
            <number>-999</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>y</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBoxPosY">
           <property name="minimum">
            <number>-999</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCommandLinkButton" name="commandLinkButton">
           <property name="text">
            <string>Launch Ray-Tracing</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="antena">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="icon.qrc">
             <normaloff>:/path4136.png</normaloff>:/path4136.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="receiver">
           <property name="text">
            <string>R</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="settingsButton">
           <property name="text">
            <string>settings</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="helpButton">
           <property name="text">
            <string>help</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="sceneRect">
        <rectf>
         <x>0.000000000000000</x>
         <y>0.000000000000000</y>
         <width>969.000000000000000</width>
         <height>505.000000000000000</height>
        </rectf>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>50</y>
      <width>211</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Transmitted power (dBm)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="powerResultSpinBox">
       <property name="minimum">
        <double>-100000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Maximum Binary debit (Mb/s)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="binaryResultsSpinBox">
       <property name="minimum">
        <double>-10000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100000000000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearWorkspaceButton">
       <property name="text">
        <string>Clear Workspace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Noto Sans';&quot;&gt;-Launch a coverage map scan. &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Noto Sans';&quot;&gt;Beware that it might take a while. The discretisation is by default 50x50&lt;/span&gt;&lt;span style=&quot; font-family:'Noto Sans'; text-decoration: underline;&quot;&gt; (fast loading n = 2/3)&lt;/span&gt;&lt;span style=&quot; font-family:'Noto Sans';&quot;&gt;, however it migh be changed in the settings menu.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Noto Sans';&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Noto Sans';&quot;&gt;- It is also adviced to reduce the amount of rebounds calculated when scanning.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Noto Sans';&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Noto Sans';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="generateCoveragePushButton">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>530</y>
      <width>180</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Coverage</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinBoxResult">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>620</y>
      <width>1009</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <double>999.990000000000009</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>1009</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance Emettor - Receptor (m)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1281</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
